tests/playbooks/e2e.yaml

```diff
@@ -99,22 +99,22 @@
           spec:
             externalTrafficPolicy: Cluster
 
-    - name: Gather facts for {{ vm_name }} VMI in default namespace
-      k8s_facts:
-        name: "{{ vm_name }}"
-        kind: VirtualMachineInstance
-        api_version: kubevirt.io/v1alpha3
-      register: vmi
+#   - name: Gather facts for {{ vm_name }} VMI in default namespace
```

Comment:
I do not like that we are commenting the code. What are the reasons that it is not working?

Reply:
Architectural differences between minikube and kubevirtci. Today I've been told of a way to reenable the test in an alternative form, which I was planning on doing in a separate PR.